themes/default/content/blog/pulumi-go-boilerplate-v2/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "The Easier Way to Create Pulumi Providers in Go"
```

Comment:
What do you think about "An Easier Way" instead? 

```suggestion
title: "An Easier Way to Create Pulumi Providers in Go"
```

Reply:
I think `The Easier Way` is a little more :hot_pepper: